x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/LocalPhysicalPlanOptimizerTests.java

```diff
@@ -746,6 +747,32 @@ public void testMissingFieldsDoNotGetExtracted() {
         assertThat(Expressions.names(fields), contains("_meta_field", "gender", "job", "job.raw", "languages", "long_noidx"));
     }
 
+    /**
+     * Expects
+     * LimitExec[1000[INTEGER]]
+     * \_ExchangeExec[[],false]
+     *   \_ProjectExec[[_meta_field{f}#9, emp_no{f}#3, first_name{f}#4, gender{f}#5, job{f}#10, job.raw{f}#11, languages{f}#6, last_n
+     * ame{f}#7, long_noidx{f}#12, salary{f}#8]]
+     *     \_FieldExtractExec[_meta_field{f}#9, emp_no{f}#3, first_name{f}#4, gen..]
+     *       \_EsQueryExec[test], indexMode[standard], query[{"match":{"first_name":{"query":"Anna"}}}][_doc{f}#13], limit[1000], sort[]
+     *       estimatedRowSize[324]
+     */
+    public void testMatchFilterPushdown() {
+        var plan = plannerOptimizer.plan("""
+            from test
+            | where first_name match "Anna"
```

Comment:
It would be better to have more complex examples here. `match` (which can be pushed down) and another construct (different than `match`) that can be also pushed down.

Reply:
added a `testMultipleMatchFilterPushdown` with multiple `where` commands using `match` and range filters